modules/ROOT/pages/6.2-release-notes.adoc

```diff
@@ -471,6 +516,14 @@ The presented list was not entirely alphabetical, however.
 
 With this update Premium plugin names are, now, correctly sorted in alphabetical order by name.
 
+[[security-fixes]]
+== Security fixes
+
+{productname} 6.2 provides fixes for the following security issues:
```

Comment:
```suggestion
{productname} 6.2 includes fixes for the following security issues:
```

Reply:
We should probably update the template as well then if we're fixing this 😃

Reply:
Fixed via 157d6e8b2181ab8a57f97c3c466d8ea01bb9c229